src/components/app/routes/data/services.js

```diff
@@ -364,6 +365,33 @@ export async function fetchSubscriptions(enterpriseUuid) {
   };
 }
 
+// Notices
+export const fetchNotices = async () => {
+  const authenticatedUser = getAuthenticatedUser();
```

Comment:
[question] Is this check on whether the user is authenticated necessary given we shouldn't be getting to this code path for unauthenticated users? For example, all the route loaders have 

```jsx
const authenticatedUser = await ensureAuthenticatedUser(requestUrl, params);
// User is not authenticated, so we can't do anything in this loader.
if (!authenticatedUser) {
  return null;
}
```

...to shortcut the execution of the remainder of the loader function given the rest of the loader function would fail without authentication.

Given that, could we always assume users are authenticated within this function?

Reply:
Good call out, I used the existing notices API without changing it so a lot of these nits are around the original implementation. Fixing it up now 👍🏽